anya/roman_numeral.rb

```diff
@@ -0,0 +1,36 @@
+class RomanNumeral
```

Comment:
Wouldn't it be more extensible to have this class represent a behavior, like `RomanNumeralConverter`, and internally it has an private enumeration `DEFAULT_DISTINCT_ROMAN_NUMERALS` and it exposes the method `numeral_to_fixnum` which takes the numeral to convert?

The initialize method should be used to inject a custom set of numerals:

````ruby
def initialize(distinct_numerals: DEFAULT_DISTINCT_ROMAN_NUMERALS)
  @distinct_numerals = distinct_numerals
end
````

````ruby
def numeral_to_fixnum(numeral_for_conversion)
  ...
end
````